### PR TITLE
Ecosystem Project Form Submission: Kabat [skip percy]

### PR DESCRIPTION
**Pull request by CharlyMartin**

### CODE REVIEW

File: src/content/ecosystem-explorer/projects/kabat.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+seo:
+  title: Kabat
+  description: SP from the beginning 
+eFIL+: from us - Motowizja, FBSNor 
+Cooperation: DeStor, Banyan, Darma, Holon 
+Based in Europe:Poland,Germany,Estonia,
+Tier4, 100%green
+---
+
+we are storage provider
```
Comment (CharlyMartin) [inline src/content/ecosystem-explorer/projects/kabat.md:34]: @dcassagnol comment:
Kabat is a storage provider on the storage network

**Comment by mirhamasala** [comment]
@CharlyMartin @matthendrian - Does this need a review still? The texts seem a little off still.

**Comment by matthendrian** [comment]
@mirhamasala yes - we're reaching out to them for some additional details and will update here once we have them.

**Comment by dcassagnol** [comment]
Can replace "About Kabat" with info from their website "Unleash the power of your data ​with Kabat! We deliver next-level ​decentralized storage and ​compute solutions, giving your ​business the speed, security, and ​scalability it deserves. It's time to ​take charge of the cloud."

**Comment by CharlyMartin** [comment]
Thanks @dcassagnol I added your suggestion. Is it good to go now?

The [description](https://github.com/FilecoinFoundationWeb/filecoin-foundation/pull/687/files#diff-044935777792a0c5f6f16c5113b4580af9a7994c26ee98ca909be76ffc22e4faR15) still feels a bit off:

> SP from the beginning eFIL+: from us - Motowizja, FBSNor Cooperation: DeStor, Banyan, Darma, Holon Based in Europe: Poland, Germany, Estonia, Tier4, 100%green

**Comment by CharlyMartin** [comment]
@matthendrian is this one good to go? The [description](https://github.com/FilecoinFoundationWeb/filecoin-foundation/pull/687/files#diff-044935777792a0c5f6f16c5113b4580af9a7994c26ee98ca909be76ffc22e4faR15) still feels a bit off, but maybe it's ok.

> SP from the beginning eFIL+: from us - Motowizja, FBSNor Cooperation: DeStor, Banyan, Darma, Holon Based in Europe: Poland, Germany, Estonia, Tier4, 100%green

Let me know :)

File: src/content/ecosystem-explorer/projects/kabat.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  - data-storage-management
+description: "SP from the beginning eFIL+: from us - Motowizja, FBSNor Cooperation: DeStor, Banyan, Darma, Holon Based in Europe: Poland, Germany, Estonia, Tier4, 100%green"
+website: https://kabat-cloud.com/
+tech:
+  - filecoin
+year-joined: 2020-12-31T23:00:00.000Z
+
+seo:
+  title: Kabat
+  description: "SP from the beginning eFIL+: from us - Motowizja, FBSNor Cooperation: DeStor, Banyan, Darma, Holon Based in Europe: Poland, Germany, Estonia, Tier4, 100%green"
```
Comment (mirhamasala) [inline src/content/ecosystem-explorer/projects/kabat.md:23]:
```suggestion
  description: "Unleash the power of your data
with Kabat! We deliver next-level
decentralized storage and
compute solutions, giving your
business the speed, security, and
scalability it deserves. It's time to
take charge of the cloud."
```

**Pull request merged**